ui/cap-react/src/antd/partials/EditableField/EditableField.js

```diff
@@ -48,12 +49,16 @@ const EditableField = ({
         data-cy="editableInput"
         autoFocus={true}
       />
-      <CheckOutlined onClick={handleApprove} style={{ color: "#389e0d" }} />
-      <CloseOutlined
-        style={{ color: "#cf1322" }}
-        onClick={handleReject}
-        data-cy="editableTitleClose"
-      />
+      {!hideButtons && (
```

Comment:
why is this needed? from where is this prop passed?

Reply:
It's not needed, I was trying something and forgot to remove this :/